app/models/concerns/derivative_intercom.rb

```diff
@@ -0,0 +1,37 @@
+# Copyright 2011-2017, The Trustees of Indiana University and Northwestern
+#   University.  Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+#   under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+#   CONDITIONS OF ANY KIND, either express or implie See the License for the
+#   specific language governing permissions and limitations under the License.
+# ---  END LICENSE_HEADER BLOCK  ---
+
+module DerivativeIntercom
+  def to_ingest_api_hash
+    {
+      label: "quality-#{quality}", # quality-low, quality-medium, quality-high
```

Comment:
Should this have a `fields` top-level key?

Reply:
These concerns are designed to build the data structure that the API expects.

Reply:
Changing the api would be too bothersome at this point.  Le sigh.